src/file/reader.rs

```diff
@@ -495,6 +531,20 @@ mod tests {
     );
   }
 
+  #[test]
+  fn test_cursor_and_file_has_the_same_behaviour() {
+    let buffer = include_bytes!("../../data/alltypes_plain.parquet");
+    let cursor = Cursor::new(buffer.as_ref());
+
+    let read_from_file = SerializedFileReader::new(File::open("data/alltypes_plain.parquet").unwrap()).unwrap();
+    let read_from_cursor = SerializedFileReader::new(cursor).unwrap();
+
+    let file_iter = read_from_file.get_row_iter(None).unwrap();
+    let cursor_iter = read_from_cursor.get_row_iter(None).unwrap();
+
+    assert!(file_iter.eq(cursor_iter));
```

Comment:
Does this compare the iterator content?

Reply:
Yes: https://play.rust-lang.org/?gist=69620a3ea343915805f0f524900883ce&version=stable&mode=debug&edition=2015